### ISSUE TITLE
Fedora Atomic support

### ISSUE THREAD

**Issue by Yattatux** [NordSecurity/nordvpn-linux#393]
Can you add `rpm-ostree` on install.sh to allow a "package layering" install in Fedora Atomic?

for now is only apt, yum, dnf and zypper.
```
install_apt
install_yum
install_dnf
install_zypper

# None of the known package managers (apt, yum, dnf, zypper) are available
echo "Error: Couldn't identify the package manager"
```

**Comment by jkemp814** [comment]
I, too, would like this added.

**Comment by ThePhatLee** [comment]
This need to be done now. Still stuck on 3.18.5 version that is latest working on Rpm-ostree systems

**Comment by blu-sea** [comment]
I too am running v3.18 (as noted above, it's the most recent version of the software that installs successfully on Fedora atomic). To be clear, I also can't upgrade to nordvpn v4.x for the same reason/bug that has been reported above. 

Additionally, I am now receiving 'warning messages' when nordvpn v3.18 CLI is launched advising that I should upgrade asap and/or that I may be forced to upgrade or nordvpn may stop working - when of course it is not possible to upgrade due to this issue.

sudo rpm-ostree install nordvpn 

results in "error: importing RPMs: Importing package 'nordvpn': Importing archive: ostree-tar: Failed to handle directory: Invalid mode 2147500525; invalid bits in mode".

If no 'easy/quick' resolution currently exists then on behalf of all atomic users, can nordvpn please continue to permit the use of v3.18 until such time as the primary cause of why v4.x fails, can be fixed? 

Whilst using the Linux GUI Network Manager and OpenVPN to access nordvpn might at first glance seem to some to be a possible workaround, frankly it is not viable for me and I suspect many other atomic users because presumably we all rely on the CLI app's unique features eg auto connect to fastest server, killswitch etc.

In summary, until this issue can be resolved, can nordvpn please not expire v3.18. Many thanks

**Comment by blu-sea** [comment]
<img width="1940" height="230" alt="Image" src="https://github.com/user-attachments/assets/0a7b4b28-56e9-4015-9a5d-28c756a4bc72" />

I have tested and can confirm that on an rpm-ostree booted system, nordvpn 4.2.3-1 (and 4.2.2) failed to install as per below:

->  sudo rpm-ostree install ./nordvpn-4.2.3-1.x86_64.rpm

error: Importing package 'nordvpn': Importing archive: ostree-tar: Failed to handle directory: Invalid mode 2147500525; invalid bits in mode"

PS thanks very much for the engagement with the atomic community on this issue, that's really appreciated. I know there are many more users who will appreciate the eventual solution also and I'm personally more than happy to continue testing for you.

**Comment by blu-sea** [comment]
PS yes, **please** keep the 3.18 package supported until the 4.x works. Many thanks!

**Comment by mariusSincovici** [comment]
Hi @blu-sea,
Version 4.3.0 was released. 
Could you please try to install it and let us know if it is working also for you?

**Comment by blu-sea** [comment]
Yes, it's working now via the work around below and thank you!

Workaround details for using install.sh on atomic rpm distros:

1. the official nordvpn linux rpm install script from https://nordvpn.com/download/linux/#install-nordvpn still fails to install on atomic linux with "error: can't create transaction lock on /usr/share/rpm/.rpm.lock (Read-only file system)
error: https://repo.nordvpn.com/gpg/nordvpn_public.asc: key 1 import failed. Apparently because the nordvpn installer script does not yet test the OS for atomic vs  dnf/yum and assumes yum/dnf will be present. To work around this:
2. manually download install.sh and read the contents and then manually download the latest nordvpn-4.x.x-x.x86.rpm file that is appropriate for your OS and hardware and save it eg currently this is nordvpn-4.3.1-1.x86.rpm
3. sudo rpm-ostree install nordvpn-4.*.rpm (this installs the nordvpn rpm as a new layer)
4. reboot so your image contains the new nordvpn layer
5. sudo systemctl enable --now nordvpnd (enable and run the nordvpnd daemon at all times)
6. sudo systemctl status nordvpnd (if no 'red', then this will show the nordvpnd daemon is now running correctly)
8. nordvpn login

Note: for more advanced atomic users running bootc  images who don't want to layer via rpm-ostree, you can copy the latest nordvpn rpm file to your custom image in your Containerfile and then during the image build run something similar to:

 dnf5 install -y /ctx/nordvpn-4.x.x-x.x86_64.rpm

and this will build your custom OS bootc image with nordvpn without the need for rpm-ostree

**Comment by blu-sea** [comment]
https://repo.nordvpn.com/yum/nordvpn/centos/x86_64/Packages/n/

**Comment by jkemp814** [comment]
Getting this message:

- nothing provides nordvpn >= 4.0.0 needed by nordvpn-gui-4.4.0-1.x86_64 from @commandline

**Comment by tecknohow** [comment]
There seems to be a regression with 4.5 (after 4.4 worked). Receiving the following error:

error: Importing package 'nordvpn': Importing archive: ostree-tar: Failed to handle directory: Invalid mode 2147500525; invalid bits in mode

**Comment by devzbysiu** [comment]
Hi @tecknohow, yes this is already raised here https://github.com/NordSecurity/nordvpn-linux/issues/1386. The fix is merged , will be released with next release
